src/natcap/invest/managed_raster/managed_raster.pxd

```diff
@@ -4,6 +4,15 @@
 from libcpp.list cimport list as clist
 from libcpp.pair cimport pair
 from libcpp.set cimport set as cset
+from libcpp.vector cimport vector
+
+cdef struct s_neighborTuple:
+    int direction
+    int x
+    int y
```

Comment:
I was just about to ask if we should consider making these `long`s, but for a raster to exceed 2^31 pixels in the x dimension, that would be a truly massive raster!  For a raster covering the whole globe, 2^31 pixels would mean a pixel width of 1.87cm along the equator.  So we'll probably never need to make this a `long`, and when we do, we'll have other problems of computability anyways.